src/org/parenthesin/components/webserver.clj

```diff
@@ -0,0 +1,19 @@
+(ns org.parenthesin.components.webserver
+  (:require [com.stuartsierra.component :as component]))
+
+(defrecord WebServer [config routes storage]
+  component/Lifecycle
+  (start [this]
+    (println ";; Starting webserver")
+    (println "port:" (get-in config [:config :webserver-port]))
+    (println "cors:" (get-in config [:config :cors-allowed-origins]))
+    (println "unsplash:" (get-in config [:config :unsplash-access-key]))
+    (assoc this :service {}))
```

Comment:
shouldn't we do something here, or this will be implemented later?

Reply:
I was planning to push the components part on this PR and later adds the webserver functions.

Then later when I need a scaffold for an empty component system I can come back to this PR 
hahaha

Reply:
nevermind, I get it :)